pkg/scheduler/apis/config/types_pluginargs.go

```diff
@@ -88,6 +88,9 @@ type NodeResourcesFitArgs struct {
 	// with "example.com", such as "example.com/aaa" and "example.com/bbb".
 	// A resource group name can't contain '/'.
 	IgnoredResourceGroups []string
+
+	// ScoringStrategy selects the node resource scoring strategy.
+	ScoringStrategy ScoringStrategy
```

Comment:
oh, change to reference, as per our last comment we wanted to revert the change, and reverting it is to use references in both the internal and external types.